lib/cdo/aws/cloud_formation.rb

```diff
@@ -23,7 +23,7 @@ class CloudFormation
     # @return [Hash]
     attr_reader :options
 
-    # Default S3 bucket for uploading temporary template files.
+    # Default S3 bucket for uploading template files.
     TEMP_S3_BUCKET = 'cf-templates-p9nfb0gyyrpf-us-east-1'
```

Comment:
This is actually the permanent location of uploaded lambda code, it's inaccurate to refer to this as temporary storage. Deleting the contents would break things.